### PR TITLE
Align to Graphics_oM `Vertex` to `RenderPoint` refactoring

### LINKED ISSUE #77: Align to Graphics_oM `Vertex` to `RenderPoint` refactoring
#### Broken rules:

See https://github.com/BHoM/BHoM/issues/1273

### PR DESCRIPTION
**Pull request by alelom**
### NOTE: Depends on 

https://github.com/BHoM/BHoM_Engine/pull/2606
   
### Issues addressed by this PR

Closes #77

### CODE REVIEW

**Comment by alelom** [comment]
@BHoMBot check required

**Comment by enarhi** [comment]
@BHoMBot check compliance

**Review by enarhi — DISMISSED** [review]
LGTM

**Comment by alelom** [comment]
@BHoMBot check documentation-compliance

**Comment by alelom** [comment]
@BHoMBot check required

**Comment by alelom** [comment]
@BHoMBot check compliance

**Review by enarhi — APPROVED** [review]
LGTM

**Comment by alelom** [comment]
As discussed with @FraserGreenroyd , code-compliance is failing on the file naming conventions, but those files where created before the check was created and the naming convention used currently has been useful for several reasons. The naming of the files will change anyway when we will do the refactoring of this toolkit. In addition, code-compliance isn't currently a required check so dispensation isn't needed.

tl;dr: ignore code-compliance failure

**Comment by alelom** [comment]
@BHoMBot check ready-to-merge

**Comment by FraserGreenroyd** [comment]
@BHoMBot check ready-to-merge

**Comment by FraserGreenroyd** [comment]
@BHoMBot check ready-to-merge
